docs/visual-basic/programming-guide/language-features/procedures/parameter-arrays.md

```diff
@@ -28,7 +28,7 @@ Usually, you cannot call a procedure with more arguments than the procedure decl
 ## Calling a ParamArray  
  When you call a procedure that defines a parameter array, you can supply the argument in any one of the following ways:  
   
-- Nothing — that is, you can omit the [ParamArray](../../../../visual-basic/language-reference/modifiers/paramarray.md) argument. In this case, an empty array is passed to the procedure. You can also pass the [Nothing](../../../../visual-basic/language-reference/nothing.md) keyword, with the same effect.  
+- Nothing — that is, you can omit the [ParamArray](../../../../visual-basic/language-reference/modifiers/paramarray.md) argument. In this case, an empty array is passed to the procedure. **NOTE: If you explicitly pass the [Nothing](../../../../visual-basic/language-reference/nothing.md) keyword, a null array is passed to the procedure and may result in a NullReferenceException if the called procedure does not check for this condition.**  
```

Comment:
I'm going to make a slightly different suggestion. The sentence on explicitly passing `Nothing` should be inline with the text, but adding the "note" word would indicate that we should apply the "note" style:

```suggestion
- Nothing — that is, you can omit the [ParamArray](../../../../visual-basic/language-reference/modifiers/paramarray.md) argument. In this case, an empty array is passed to the procedure. If you explicitly pass the [Nothing](../../../../visual-basic/language-reference/nothing.md) keyword, a null array is passed to the procedure and may result in a NullReferenceException if the called procedure does not check for this condition.
```